Zend/tests/traits/constant_002.phpt

```diff
@@ -0,0 +1,29 @@
+--TEST--
+Defining a constant in both trait and its composing class with the same name, visibility, finality and value is allowed
+--FILE--
+<?php
+
+trait TestTrait1 {
+    public const A = 42;
+}
+
+trait TestTrait2 {
+    use TestTrait1;
+    public const A = 42;
+}
+
+trait TestTrait3 {
+    use TestTrait2;
+    public const A = 42;
+}
+
+class ComposingClass {
+    use TestTrait1;
+    use TestTrait3;
+    public const A = 42;
+}
```

Comment:
Isn't it better to test that the composing class works by using two *unrelated* traits (i.e. a trait A and B where B doesn't use A)? Or is there already such a test case?

Reply:
Done.